config.json

```diff
@@ -263,7 +263,13 @@
       "core": true,
       "difficulty": 4,
       "slug": "rotational-cipher",
-      "topics": null,
+      "topics": [
+        "integers",
+        "ascii",
+        "chars",
+        "strings",
+        "cryptography"
```

Comment:
Great suggestions! :) I'm not sure about `ascii` and `chars` though. I think `chars` is pretty much covered by `strings` so I don't think we need it as a separate topic. I'm also not sure `ascii` is that relevant to this exercise since you don't really need to know anything about `ascii` to solve the exercise as far as I'm aware?